historical_sst_sic/create_artifact.jl

```diff
@@ -17,17 +18,84 @@ else
     mkdir(output_dir)
 end
 
-for (path, url) in (
-    SIC_FILE_PATH => SIC_FILE_URL,
-    SST_FILE_PATH => SST_FILE_URL
-    )
+for (path, url) in
+    (SIC_FILE_PATH => SIC_FILE_URL, SST_FILE_PATH => SST_FILE_URL)
     if !isfile(path)
         @info "$path not found, downloading it (might take a while)"
         downloaded_file = Downloads.download(url)
         Base.mv(downloaded_file, path)
-        Base.cp(path, joinpath(output_dir, basename(path)))
     end
+    Base.cp(path, joinpath(output_dir, basename(path)))
 end
 
 @info "Data file generated!"
 create_artifact_guided(output_dir; artifact_name = basename(@__DIR__))
+
+function thin_artifact(
```

Comment:
it would be good to add a document explaining what this function does

Reply:
Added